dask_ml/model_selection/_hyperband.py

```diff
@@ -109,6 +113,36 @@ class HyperbandSearchCV(BaseIncrementalSearchCV):
         optimal. ``aggressiveness=4`` has higher confidence that is likely
         suitable for initial exploration.
 
+    explore : int, bool, default=False
+        Run a search geared towards an initial exploratory search if this
+        parameter is specified. ``explore`` is typically specified when
+        not much is known about the hyperparameters and/or model.
+
+        If ``explore`` is a bool, run a search aimed at finding the same
+        validation accuracy as Hyperband with ``explore=False`` but with
+        less computation.
+        If ``explore`` is an integer, repeat the most exploratory bracket
```

Comment:
What does "exploratory" mean in this context?

Reply:
It means the bracket that samples the largest number of models.